compiler/rustc_codegen_ssa/src/back/archive.rs

```diff
@@ -52,10 +52,30 @@ pub trait ArchiveBuilder<'a> {
 
     fn build(self) -> bool;
 
+    fn sess(&self) -> &Session;
+
+    fn create_dll_import_lib(
+        sess: &Session,
```

Comment:
Can you add a doc-string describing what `create_dll_import_lib()` does and why it does not take `&mut self` like the other methods in this trait.